eventyay_business/models.py

```
@@ -172,3 +172,70 @@ class Meta:
+    )
+    stripe_subscription_id = models.CharField(
+        max_length=255, blank=True, null=True, verbose_name=_("Stripe subscription ID")
+    )
+    configuration_snapshot = models.JSONField(
+        default=dict, blank=True, verbose_name=_("Configuration snapshot")
+    )
+    created_at = models.DateTimeField(auto_now_add=True, verbose_name=_("Created at"))
+    updated_at = models.DateTimeField(auto_now=True, verbose_name=_("Updated at"))
+
+    class Meta:
+        ordering = ["-starts_at", "-id"]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)